### ISSUE TITLE
error on PKG_RIF-INSTALLER.pkg_signed.pkg 

### ISSUE THREAD

**Issue by char115** [PS3Xploit/PS3xploit-resigner#22]
i have a complaint of this error always keep showing on all version its simular to the ones you faced with someone but this is new i have no idea it keeps showing.. is it bc of the antivirus? missing a file? or i place on  wrong order? please help me i want to play my childhood game in a ps2 era also heres the photo from a past problem

note to self the link isnt working so thats why im making this

![37571119-031ed374-2abe-11e8-802b-12f1cdcf1c0d](https://user-images.githubusercontent.com/40615318/41945012-8d40884e-7970-11e8-841d-cb087c4842c7.png)

**Comment by kingvz** [comment]
I have the same problem. Any solution? :(

**Comment by KittyApoc** [comment]
Check to see if you still have the app "pkg_exdata" in the source/tools/ps3py_exe folder. I had a similar issue but it was not the exact same. I had all the input files correct, but every time I tried to run it, I ran into this error:
![command](https://user-images.githubusercontent.com/41137091/42614787-3a7e7146-855c-11e8-953c-b61d1e6a5b2f.JPG)
I saw that looking through the files, that the pkg_exdata file was gone. My antivirus flagged it as a virus and removed it, and after redownloading the file it would prevent me from even moving it. Once I disabled the antivirus, I moved it back into the ps3py_exe folder and ran resign_windows again, and it worked. Its not the exact same issue, but it looks similar to the one I had so it may be worth a try.
